### PR TITLE
Remove redaxscript from Wikis

### PR DESCRIPTION
**Pull request by ardichoke**
redaxscript is a CMS, not a Wiki. It is already present in the CMS section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -463,7 +463,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [ProcessWire](https://processwire.com/) - ProcessWire is an open source content management system (CMS) and web application framework aimed at the needs of designers, developers and their clients. ([Source Code](https://github.com/ryancramerdesign/ProcessWire)) `MPLv2` `PHP`
   * [PropertyWebBuilder](http://propertywebbuilder.com) - The ultimate Ruby on Rails engine for creating real estate websites. ([Demo](https://propertywebbuilder.herokuapp.com), [Source Code](https://github.com/etewiah/property_web_builder)) `MIT` `Ruby`
   * [Publify](http://publify.co/) - A simple but full featured web publishing software. ([Demo](http://demo.publify.co/), [Source Code](https://github.com/publify/publify)) `MIT` `Ruby`
-  * [REDAXO](https://www.redaxo.org) - A simple, flexible and useful content management system (documentation only available in German). ([Source Code](https://github.com/redaxo/redaxo)) `MIT` `PHP`
+  * [cO](https://www.redaxo.org) - A simple, flexible and useful content management system (documentation only available in German). ([Source Code](https://github.com/redaxo/redaxo)) `MIT` `PHP`
```
Comment (nodiscc) [inline README.md]: Why did you change this to `c0` ?

File: README.md [hunk 2]
```diff
@@ -1200,7 +1200,6 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
   * [PmWiki](http://www.pmwiki.org) - Wiki-based system for collaborative creation and maintenance of websites. `GPLv3` `PHP`
   * [Raneto](http://raneto.com/) - Raneto is an open source Knowledgebase platform that uses static Markdown files to power your Knowledgebase. `MIT` `NodeJS`
   * [Realms](http://realms.io/) - A git-backed wiki inspired by Gollum. ([Source Code](https://github.com/scragg0x/realms-wiki)) `GPLv2` `Python`
-  * [Redaxscript](http://redaxscript.com/) - Ultra lightweight website engine. ([Source Code](https://github.com/redaxmedia/redaxscript)) `GPLv3` `PHP`
```
Comment (nodiscc) [inline README.md]: Redaxscript is indeed a CMS and should be moved to the appropriate section.
But Redaxo is another project, so there is no duplicate here.

Can you move redaxscript to the CMS section? (instead of removing it)

Thanks

**Comment by ardichoke** [comment]
Sorry about the REDAXO part, that was accidental on my part and I missed it before committing.

As for redaxscript, per my original commit, it already is present in the CMS section... it was just ALSO present in the Wiki section.
